gym/lib/gym/error_handler.rb

```diff
@@ -259,7 +262,7 @@ def print_environment_information
               # As seen above, there is obviously a mismatch, the user selected an App Store
               # profile, but the export method that's being passed to Xcode is "enterprise"
 
-              next if matching_type.to_s == selected_export_method
+              break if matching_type.to_s == selected_export_method
```

Comment:
I think this should be `next` and not `break`, right?

Reply:
The `next` or `break` makes sense when the profile name matches two different export types.

When profile name is `Ad-Hoc Development` and export_method is `ad-hoc`, the printed messages are as follows.

#### `next`

```
[02:32:17]: There seems to be a mismatch between your provided `export_method` in gym
[02:32:17]: and the selected provisioning profiles. You passed the following options:
[02:32:17]:   export_method:      ad-hoc
[02:32:17]:   Bundle identifier:  my.org.app
[02:32:17]:   Profile name:       Ad-Hoc development
[02:32:17]:   Profile type:       development
````

1. `Ad-Hoc` matches `ad-hoc` and is equal to the export method, then `next` to check subsequent types
2. `Development` matches `development`, then print the message

#### `break`

The mismatch message does not printed.

1. `Ad-Hoc` matches `ad-hoc` and is equal to the export method, then `break`

----
I think that there is a high possibility that the detected profile type after the `next` is incorrect, so I changed it to `break`. But of course, I'll fix it to `next` if you think it is better to print the message.